client/api/noaExample.js

```diff
@@ -0,0 +1,14 @@
+import request from 'superagent';
+import { handleSuccess, handleError } from '_utils/api';
+
+var msg = '';
```

Comment:
Hi Noa
Usualy in such cases, we save the value of the response in a **state**.
Either a state inside the comopnent, with useState, or a global state, with redux.
Last tuesday I showed you an example with state inside the component.